core/trino-main/src/main/java/io/trino/server/protocol/Query.java

```diff
@@ -531,7 +532,6 @@ private synchronized QueryResultRows removePagesFromExchange(QueryInfo queryInfo
         QueryResultRows.Builder resultBuilder = queryResultRowsBuilder(session)
                 // Intercept serialization exceptions and fail query if it's still possible.
                 // Put serialization exception aside to return failed query result.
-                .withExceptionConsumer(this::handleSerializationException)
```

Comment:
Is this method `handleSerializationException` still needed? I don't see any new usage